src/data/faq.json

```diff
@@ -2969,24 +2978,30 @@
                                 "anchor": "check_in_luca_how",
                                 "active": false,
                                 "textblock": [
-                                    "The Corona-Warn-App can now scan QR codes created for event registration with the luca app and perform its own check-in process. Luca cannot scan QR codes created by the Corona-Warn-App."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. However, luca QR codes can still be scanned with the Corona-Warn-App.<hr>",
+                                    "The Corona-Warn-App can scan QR codes created for event registration with the luca app and perform its own check-in process. The luca app could have not scanned QR codes created by the Corona-Warn-App."

```

Comment:
"The luca app could have not scanned QR codes ..." =>
"The luca app cannot scan QR codes ..."

otherwise you have to explain about the use of the "could have not" tense

Reply:
I'd suggest to just drop all these "could have not if luca still offered contact tracing". They do no longer, so this information is irrelevant for the FAQ.